packages/unit-testing/stubs/workbook/CalculatedMeasure.js

```diff
@@ -0,0 +1,49 @@
+define(['./Expression'], function () {
```

Comment:
Missing Expression param in function
`define(['./Expression'], function (Expression) {`

Reply:
Corrected